blog/_posts/2017-09-18-story-of-an-issue.md

```diff
@@ -0,0 +1,100 @@
+
+#### Eat ice cream
+
+while celebrating merged PR.  
+
+## What did the story of an issue teach us so far?
+
+Here are some lessons, big and small, we've learned from this roller coaster called SoC:
+
+**Know your study materials:** Find your favorite source for study materials: whether you like to watch, read or practice, whether you prefer it on-line or hard-copy, it is very efficient to have a go-to learning sources.
+
+**Mix and match:** Using diverse ways of learning, which means a combination of watching, reading, exercising and working on the source code, is the most useful way to tackle new topics. It might seem obvious, but it's still an important lesson for us.
+
+**Master git:** Pretty self explanatory, but we can't stress enough how helpful it was to understand our git-flow.
+
+**Keep a notebook:** Use the traditional pen and paper to keep track of new terms and concepts, which at first glimpse could be quite overwhelming and even intimidating. It's also a great place to sketch and doodle.
+
+**Try not to be afraid** to make mistakes, play with your code, express your opinions and make your own decisions. After 10 weeks of working regularly with 6 coaches, each one with their own way of solving problems, we realized the only thing we have left to do is decide which way works best for us.
+
+**Try not to be shy** and ask for help and explanations. Experience shows it often works for the best.
+
+**Learn from your code reviews:** See "Code review" above :).
+
+**We are in love with diaspora\*:** The complexity of the code and the involvement of the community have made this summer very challenging and rewarding. Check the [project on github](https://github.com/diaspora/diaspora), maybe you would want to contribute as well.
+
+**Remember to celebrate:** Did you open a PR, did your code get merged, did you just finish refactoring a beautiful piece of code? Great! Celebrate and go eat some ice cream.
+
+## Thanks
+
+This amazing summer would not be amazing without our team and the help of other supportive people:
+
+* Our team's coaches: **Chiara, Dani, David, Glauber, Jano** and **Laura**, thank you so much for all you help, time, energy, patience, white-board sessions, source-code sessions, exercises and so much more.
+* Other coaches: **Alon, Andrey, Liron** and **Remi** for helping on specific topics and substituting for coaches on vacation.  
+* Our supervisor: **Fanny**, thank you for caring for our well-being throughout the summer and always being there for questions.
+* Our mentor: **Lisa**, thanks for your availability for questions and tips about diaspora\*.
+* Diaspora community: Thank you for being so responsive and helpful, and for keeping such a pleasant communication with your contributers. Special thanks for members who reviewed patiently and commented on our code- **@Flaburgan, @SuperTux88** and **@svbergerem**.
```

Comment:
Do you mean to attach links for Flaburgan, SuperTux88 and svbergerem? Then you can use markdown style or you can just mention their names without @ . :)